controls/combobox/overview.md

```diff
@@ -8,15 +8,15 @@ slug: combobox-overview
 
 # .NET MAUI ComboBox Overview
 
-**Telerik UI for .NET MAUI ComboBox** allows users to select item/items from a drop down list of items. The control has a number of features such as searching, single and multiple selection, flexible styling API, dropdown customizations and more.  
+The Telerik UI for .NET MAUI ComboBox allows users to select item/items from a drop down list of items. The control has a number of features such as searching, single and multiple selection, flexible styling API, dropdown customizations and more.  
 
-![ComboBox Overview](images/combobox-overview.png "ComboBox Overview")
+![.NET MAUI ComboBox Overview](images/combobox-overview.png "ComboBox Overview")
 
 ## Key Features
 
 * [Editable and NonEditable mode]({%slug combobox-editmode-and-search%}) – ComboBox supports both editable and noneditable state. When the control is in edit mode searching can be performed. 
```

Comment:
Use an em dash as a separator in the definition list. This comment is valid for all list items that follow.
```suggestion
* [Editable and NonEditable mode]({%slug combobox-editmode-and-search%})—ComboBox supports both editable and noneditable state. When the control is in edit mode searching can be performed. 
```